docroot/themes/humsci/humsci_basic/src/scss/components/_secondary-nav.scss

```diff
@@ -118,7 +118,11 @@
 
     &::before {
```

Comment:
can you add a note to refactor this code at some point with code in the main nav?